README.md

```
@@ -310,15 +383,17 @@ The ReportPortal MCP server provides a comprehensive set of capabilities for int
 - Get launch details by name or ID
 - Force-finish running launches
 - Delete launches
-- Run automated analysis (auto analysis, unique error analysis) on launches
+- Run automated analysis (auto analysis, unique error analysis, quality gate) on launches
 
 ### Test Item Analysis
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)